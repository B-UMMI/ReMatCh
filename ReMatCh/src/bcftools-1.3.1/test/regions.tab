1	3062915	3062915
1	3106154	3106154
1	3157410	3157410
1	3162006	3162006
1	3184885	3184885
2	3199815	3199815
3	3212016	3212016
3	3212036	3212036
