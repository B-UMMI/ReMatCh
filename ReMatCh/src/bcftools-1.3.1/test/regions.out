1 3062915 GTT,G
1 3062915 G,T
1 3106154 CA,C
1 3106154 C,T,CT
1 3157410 G,A
1 3162006 G,A
1 3184885 T,TA
2 3199815 C,T
3 3212016 C,A
3 3212036 C,A
